interfaces/cython/cantera/onedim.py

```diff
@@ -299,6 +300,147 @@ def write_csv(self, filename, species='X', quiet=True):
         if not quiet:
             print("Solution saved to '{0}'.".format(filename))
 
+    def to_solution_array(self):
+        """Return the solution vector as a Cantera SolutionArray object.
+
+        The SolutionArray has the following ``extra`` entries:
+         * `grid`: grid point positions along the flame [m]
+         * `velocity`: normal velocity [m/s]
+         * `gradient`: tangential velocity gradient [1/s] (if applicable)
```

Comment:
`velocity_gradient` imho isn't great either as it could be mistaken for the normal velocity gradient. I initially had `tangential_velocity_gradient`, which is a mouthful. Any alternatives??

Reply:
Resolving this via the suggestion `vGradient`; this follows the CamelCase precedent `eField`.